test/suites/scheduling/suite_test.go

```diff
@@ -168,6 +168,28 @@ var _ = Describe("Scheduling", Ordered, ContinueOnFailure, func() {
 			env.EventuallyExpectHealthyPodCount(labels.SelectorFromSet(deployment.Spec.Selector.MatchLabels), int(*deployment.Spec.Replicas))
 			env.ExpectCreatedNodeCount("==", 1)
 		})
+		It("should support well-known labels for local instance storage", func() {
+			selectors.Insert(v1.LabelInstanceLocalStorage) // Add node selector keys to selectors used in testing to ensure we test all labels
+			deployment := test.Deployment(test.DeploymentOptions{Replicas: 1, PodOptions: test.PodOptions{
+				NodePreferences: []corev1.NodeSelectorRequirement{
```

Comment:
Can these values ever be zero, or would the labels not exist in that case? If they can be zero, should we filter those values out in the instance type provider? I think users are used to being able to use the `Exists` operator to check for the existence of a resource, this would break that assumption.

Reply:
They can't be zero. If the value is set, there will be a value exposed for it (see what's exposed in the instance types file that we generate with this change)